### PR TITLE
Update imaging-edge from 2.0.2_1911a,DqbvQuTn2u to 1_0_02,2BqD6b8EKL

### PR DESCRIPTION
**Pull request by suschizu**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by core-code** [comment]
i see this URL in the redirect?

https://di.update.sony.net/NEX/NePIDhlELg/IE210_2003a.dmg

**Comment by digitalqualms** [comment]
Looks like ied_1_0_02.pkg is the new package in ied_2.0.2_1911a.dmg and that's why the build is failing.

That installs /Applications/Imaging Edge Desktop.app which then has a download button that gets https://support.d-imaging.sony.co.jp/disoft_DL/imagingedge_DL/mac?fm=dt

Which redirects and downloads IE210_2003a.dmg which contains IE_INST.pkg which finally installs /Applications/Imaging Edge/*.app.

I'm not sure what the right thing is for this cask.

File: Casks/imaging-edge.rb [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-  appcast 'https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://support.d-imaging.sony.co.jp/disoft_DL/imagingedge_DL/mac?fm=en',
+  url "https://di.update.sony.net/NEX/#{version.after_comma}/ied_#{version.before_comma.dots_to_underscores}.dmg"
+  appcast 'https://macupdater.net/cgi-bin/check_urls/check_url_redirect.cgi?url=https://support.d-imaging.sony.co.jp/disoft_DL/desktop_DL/mac?fm=en',
           configuration: version.after_comma
   name 'Sony Imaging Edge'
-  homepage 'https://support.d-imaging.sony.co.jp/app/imagingedge/'
+  homepage 'https://imagingedge.sony.net/en-us/ie-desktop.html'
 
-  pkg 'IE_INST.pkg'
+  pkg "ied_#{version.before_comma}.pkg"
```
Comment (miccal) [inline Casks/imaging-edge.rb:11]: `pkg "ied_#{version.before_comma.dots_to_underscores}.pkg"`

**Comment by core-code** [comment]
```
installer: The install failed (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.)
```

**Comment by core-code** [comment]
going to merge this now, we can revisit if there are real problems apart from travis

**Pull request merged**